dockers/docker-database/Dockerfile.j2

```diff
@@ -37,6 +37,7 @@ COPY ["docker-database-init.sh", "/usr/local/bin/"]
 COPY ["database_config.json.j2", "/usr/share/sonic/templates/"]
 COPY ["database_global.json.j2", "/usr/share/sonic/templates/"]
 COPY ["files/supervisor-proc-exit-listener", "/usr/bin"]
+COPY ["files/sysctl-net.conf", "/etc/sysctl.conf"]
```

Comment:
What is the reasoning behind naming the file `sysctl-net.conf` in the repo, since you're copying it as `sysctl.conf`?

Reply:
@jleveque yes

Reply:
Sorry, but I was asking for your reasoning behind naming it `sysctl-net.conf` in the repo.

Reply:
@jleveque since it only have sysctl.net.* (network related) parameters. so named it that way.
There are other non network related kernel parameters that are set directly in build_debian.sh

Reply:
I see. Thanks. In that case, and if we believe we may potentially break out the other non-network related sysctl parameters into a separate file in the future, maybe you should copy this file without changing the name, as follows:

```
COPY ["files/sysctl-net.conf", "/etc/sysctl.d/"]
```

Reply:
sure, ok, in that case we have to execute: "sysctl -p -e sysctl-net.conf". (as default name is sysctl.conf)
will make change accordingly.

Reply:
Does it not automatically load /etc/sysctl.conf as well as any files in /etc/sysctl.conf?

Reply:
Have not tried but based on comment in https://linux.die.net/man/8/sysctl
-p
Load in sysctl settings from the file specified or /etc/sysctl.conf if none given.

Reply:
Maybe try it out first? As I stated, this is typical Linux behavior. If it does not work as I expect it to, then I prefer not changing the command to `sysctl -p -e sysctl-net.conf`. But I believe it should work. I want to say the man page is incomplete.

Reply:
@jleveque it works fine. we have to use sysctl --system option.  Updated accordingly.

Please review.